release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3488,3 +3488,23 @@ Now these IP addresses are periodically collected and made available to be reall
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-56"]
+=== RHBA-2022:0867 - {product-title} 4.6.56 bug fix and security update
+
+Issued: 2022-03-23
+
+{product-title} release 4.6.56, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0867[RHBA-2022:0867] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0866[RHSA-2022:0866] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6833311[{product-title} 4.6.56 container image list]
+
+[id="ocp-4-6-56-bug-fixes"]
+==== Bug fixes
+ * Previously, an error between {rh-openstack} credentials secret creation and `kube-controller-manager` startup prevented {rh-openstack} credentials from configuring properly. Consequently, `LoadBalancer` services were not created in {rh-openstack}. This updates fetches the {rh-openstack} credentials when `kube-controller-manager` starts. As a result, {rh-openstack} secret credentials now initialize properly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2059677[*BZ#2059677*])
```

Comment:
first instance of openstack needs to be {rh-openstack-first}

Reply:
Done! :)